parsl/dataflow/dflow.py

```diff
@@ -1410,18 +1409,24 @@ class DataFlowKernelLoader:
     def clear(cls) -> None:
         """Clear the active DataFlowKernel so that a new one can be loaded."""
         cls._dfk = None
+<<<<<<< HEAD
+    self.cleanup()
 
+=======
+        self.cleanup()
+>>>>>>> c4f3087c (fix for 3104)
```

Comment:
This is an uncleaned up merge conflict.  If this is a new concept, one phrase for the search engines is `merge conflict`.